CustomOperator/README.md

```diff
@@ -0,0 +1,234 @@
+This document describes the required steps for extending TorchScript with a custom operator, exporting the operator to ONNX format, and adding the operator to ONNX Runtime for model inference.

```

Comment:
Title?

Reply:
Thanks, I updated the title and added the link to main page.
Please let me know if you think we can link this in other ONNX/ORT repos.